src/main/java/org/mskcc/cbio/oncokb/service/dto/companyadditionalinfo/CompanyLicense.java

```diff
@@ -0,0 +1,34 @@
+package org.mskcc.cbio.oncokb.service.dto.companyadditionalinfo;
+
+import java.io.Serializable;
+import java.time.LocalDate;
+
+public class CompanyLicense implements Serializable {
+  private LocalDate activation;
```

Comment:
We have been suing Instant for dates, let's do that.

Reply:
I'm using LocalDate since it represents a date only. I'm assuming we don't want to deal with time zones?

Reply:
We've been using instant with timezone and map it to local in the frontend.